app/controllers/repositories_controller.rb

```diff
@@ -242,6 +242,9 @@ def import_records
           import_records = repostiory_import_actions
           status = import_records.import!
 
+          log_activity(:import_inventory_items,
```

Comment:
Do we care if the import was successfully finished? Or just call `log_activity` always?